_posts/events/2015-06-04-mwe.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+categories: [event]
+event_name: "The 11th Workshop on Multiword Expressions (MWE 2015)"
+event_url: "http://multiword.sourceforge.net/mwe2015"
+event_location: "Denver, Colorado, USA"
+event_submission_date: "2015-03-15"
+host_name: "NAACL 2015"
+host_url: "http://naacl.org/naacl-hlt-2015/"
+siglex_endorsed: yes
+---
+The 11th Workshop on Multiword Expressions (MWE 2015)
+
+NAACL 2015 (Denver, Colorado, USA), June 4, 2015
```

Comment:
You probably want two spaces at the end of this line so that the URL shows up on its own line.

Reply:
Done